core/domain/exp_services.py

```diff
@@ -1081,6 +1067,15 @@ def compute_exploration_contributors_summary(exploration_id):
                 'version_number']
         else:
             current_version -= 1
+
+    contributor_ids = list(contributors_summary)
+    # Remove IDs that are deleted or do not exist.
+    users_settings = user_services.get_users_settings(contributor_ids)
```

Comment:
Why do we have ids which doesn't exists or deleted, should we remove them through one-off jobs?

Reply:
When we delete a user as part of wipeout then we regenerate the summary and we need to get rid of the newly deleted user.

Reply:
Is there any issue with having a separate function that takes specific user_id and removes it?

Reply:
I used this approach before (or similar I passed the ID of the user that should be deleted from the summary), but with Sean we decided taht it would be better to check that the actual user exists.

Reply:
Hi @vojtechjelinek -- I'm having a chat with @DubeySandeep and I think you misinterpreted his comment. He is asking, why can there be "None" elements in contributor_ids. In other words, why don't we handle that case around line 1058 above, with an explanation given for why snapshot_metadata['committer_id'] can be None? 

FYI @DubeySandeep is also wondering why such IDs could be none or non-existent in the model -- wouldn't they always be valid regular or pseudonymized IDs?